libasn1fix/asn1fix_cws.c

```diff
@@ -307,6 +307,8 @@ _asn1f_assign_cell_value(arg_t *arg, struct asn1p_ioc_row_s *row, struct asn1p_i
 			FATAL("Cannot find %s referenced by %s at line %d",
 				p, arg->expr->Identifier,
 				arg->expr->_lineno);
+			asn1p_ref_free(ref);
```

Comment:
1. Yes, put `free(p)` above `WARNING("....")` make it more clear to reader.
2. Conflict in code is inevitable, since making feature can work independently is different goal to make them co-workable. 

In this case, I can use only one pull request with `fix 147 commit` then `fix memory leak commit`. But if people decide only accepting fix memory feature ? Of course he can cherry-pick the latter one, but there still be code conflict exist. 

I do't know how to treat this dilemma. Any advice ?